scripts/install

```diff
@@ -38,6 +38,9 @@ ynh_app_setting_set --app $app --key nginx_status_port --value $nginx_status_por
 # STANDARD MODIFICATIONS
 #=================================================
 
+# Creating the folder
+mkdir -p /var/lib/monitorix/www/imgs
```

Comment:
Uuuh that line looks like it's coming from nowhere though ... can you elaborate on why this is needed ?